pages/queue.js

```diff
@@ -56,6 +58,9 @@ class Queue extends React.Component {
     return (
       <Layout>
         <Container fluid>
+          <h5>
```

Comment:
Try splitting into two lines, maybe with a combination of `<h3>`/`<h5 className="text-muted">`, would look even better with the location icon that's used on the queue cards on the homepage!